readthedocs/core/static-src/core/js/doc-embed/search.js

```diff
@@ -0,0 +1,127 @@
+/*
+ * Sphinx search overrides
+ */
+
+var rtddata = require('./rtd-data');
+
+
+function init() {
+    var data = rtddata.get();
+    attach_elastic_search_query(data);
+}
+
+
+/*
+ * Search query override for hitting our local API instead of the standard
+ * Sphinx indexer. This will fall back to the standard indexer on an API
+ * failure,
+ */
+function attach_elastic_search_query(data) {
+    var project = data.project,
+        version = data.version,
+        language = data.language || 'en',
+        api_host = data.api_host,
+        subprojects = data.subprojects || {},
+        canonical_url = data.canonical_url || "/";
+
+    var query_override = function (query) {
+        var search_def = $.Deferred(),
+            search_url = document.createElement('a');
+
+        search_url.href = api_host;
+        search_url.pathname = '/api/v2/search';
+        search_url.search = '?q=' + query + '&project=' + project +
+            '&version=' + version + '&language=' + language;
```

Comment:
Yeah, for now they are just their own projects. It would be best if a super project with language `fr` would search for subprojects with language `fr` as well. Definitely part of a larger refactor though.